Library/Homebrew/os/mac/version.rb

```diff
@@ -24,6 +24,16 @@ class Version < ::Version
         yosemite:    "10.10",
       }.freeze
 
+      # TODO: bump version when new macOS is released or announced
+      # and also update references in docs/Installation.md and
+      # https://github.com/Homebrew/install/blob/HEAD/install.sh
+      MACOS_NEWEST_UNSUPPORTED = "12.0"
+
+      # TODO: bump version when new macOS is released and also update
+      # references in docs/Installation.md and
+      # https://github.com/Homebrew/install/blob/HEAD/install.sh
+      MACOS_OLDEST_SUPPORTED = "10.14"
```

Comment:
I'd consider either inlining these or making it a private constant. Don't feel strongly, though.

Reply:
These were previously inlined (string literals in the methods) but the idea for making them constants was that it would allow us to keep them near `SYMBOLS`, which also needs to be updated when a new macOS version appears. I would prefer to keep them as constants for this reason but I'm happy to make them private.

Reply:
@samford Gotcha, that makes sense to me to keep them near `SYMBOLS` then as constants 👍🏻